src/view/renderer.js

```diff
@@ -171,6 +171,10 @@ export default class Renderer {
 	render() {
 		let inlineFillerPosition;
 
+		// Remove unneeded fake selection container from the DOM so it will not affect the DOM nodes diffing.
```

Comment:
You say it's unneeded, but the changes might be completely unrelated to the selection (e.g. collab changes).

Reply:
It's a draft :stuck_out_tongue: 

I'll update the PR with previous solution as I thought on it over night.

Reply:
The other change I'm going to add & test is to exclude the FSC from the differ because it doesn't belong to the view. Additionally the FSC rendering is done afterwards after the differ. The FSC shouldn't affect the differ.

ps.: AFAICS the differ always marked the FSC as "removed" node.